SpheriCart.jl/Project.toml

```
@@ -0,0 +1,19 @@
+name = "SpheriCart"
```

nitpick: I've always considered the package to be a single word, not a concatenation of two of them

```suggestion
name = "Sphericart"
```

---

I'll only accept that change under pressure. It is clearly a concatenation of words and hence the C should be capitalised. It is not an "Eigenname".